### ISSUE TITLE
Paper results and plots

### ISSUE THREAD

**Issue by Garrafao** [Garrafao/LSCDBenchmark#85]
We need:

- [x] wic plots for all datasets and models (as defined in #57). Please push experiment script and link here.
- [x] change_graded plots for all datasets and models with apd_compare_all, jsd_dot_all, diasense_all, jsd_dot_all_downsampled (as defined in #57). Please push experiment script and link here.
- [ ] plots comparing apd_compare_all performance on previous versions of some datasets with current version. Let's do this for all 4 WiC models (DM1, DM2, XL-LEXEME, XLM-R). We compare: 
    - DWUG DE 1.1.0/2.3.0/3.0.0
    - DWUG EN 1.0.0/2.0.1/3.0.0
    - DWUG SV 1.0.0/2.0.1/3.0.0, 
    - DiscoWUG 1.1.1/2.0.0
    - DWUG ES 4.0.0/4.0.2
- [ ] For DWUG ES we need an additional plot comparing apd_compare_all performance on version 4.0.0 and 4.0.2 on only the test portion (basically all the lemmas present in V3.0.0). Let's do this for all 4 WiC models (DM1, DM2, XL-LEXEME, XLM-R)
- [ ] For DWUG DE 3.0.0 we need a plot comparing apd_compare_all performance with all 4 WiC models with `dataset/preprocessing=toklem,raw,tokenization,normalization,lemmatization`. @sachinn12 Could you please ASAP paste the parameters you use for `dataset/spelling_normalization` and `dataset/preprocessing` in the other experiments here? Preferably push your scripts under `experiments`.
- [ ] *update*: please run apd_compare_all with the two new DM checkpoints (MCL and MCLen) on DWUG DE/EN/SV 3.0.0 and DWUG DE/EN/SV Resampled 1.0.0. Please start experiments asap.

@sachinn12 Could you please provide your experiment scripts under `experiments` ASAP?
@sachinn12 Is it possible to prioritize these tasks to have results ready by Thursday? Please update this issue regularly so I can spot possible problems early?

**Comment by sachinn12** [comment]
You seem to be using mostly no spelling normalization, check [here](https://github.com/Garrafao/LSCDBenchmark/blob/ba0ee6ef5c78f02b4d3dbc82d07db376d359e30e/experiments/diasene_all.sh#L87), but sometimes you do, check [here](https://github.com/Garrafao/LSCDBenchmark/blob/ba0ee6ef5c78f02b4d3dbc82d07db376d359e30e/experiments/apd_compare_all.sh#L133). What's the logic? And with which parameters were the plots above produced?[ ] 

=> All the results above use 'none' for normalization. I have many script files, and while combining them, I may have copied one with normalization set to 'english' by mistake. But the plot only includes results with 'none'.

**Comment by Garrafao** [comment]
The preprocessing plot above contaibs results for DWUG EN, but we need DWUG DE. Also please put models in the same order as on the other plots.

**Comment by Garrafao** [comment]
Could you please put then LSCD model on the plot? Apd_compare_all I guess.

**Comment by nvanva** [comment]
After looking at the results, I would suggest add one more DM model that is the SOTA DM model on RuShiftEval. We now have the winning models from the Spanish shared task finetuned for Spanish, and a couple of basic models not finetuned to any specific language, but the winning model for the Russian shared task fine-tuned for Russian is missing for some reason. @Garrafao Would it be interesting to add its results? If so I can add it.

**Comment by Garrafao** [comment]
Yes, please add it! For the for the experiments we should limit the number of models to test though.

**Comment by Garrafao** [comment]
Merged with #94 .
